### DISCUSSION TITLE
manageAppVersionAndBuildNumber option doesn't increment build number

### DISCUSSION THREAD

**Discussion by patricks** [fastlane/fastlane#20031] [Q&A]
Hi,

Since Xcode 13 there is a new `manageAppVersionAndBuildNumber` export option which should automatically manage the build number when uploading to App Store Connect.

<img width="765" alt="Screen Shot 2022-03-04 at 11 13 16" src="https://user-images.githubusercontent.com/132167/156744712-06d8be9a-a03b-45bf-890a-57422b7a814f.png">

This works fine if I try to upload the build directly via Xcode. If I try to upload the TestFlight build via fastlane I always geht the following error:

```sh
ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '218' for version number '9.1.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
```

I also tried to modify my `build_ios_app` command, afaik this shouldn't be needed since this feature is enabled by default.

```ruby
build_ios_app(configuration: "Public-Release", export_method: "app-store", export_options: { manageAppVersionAndBuildNumber: true })
```

the upload command looks like this:

```ruby
upload_to_testflight(skip_submission: true, skip_waiting_for_build_processing: true, api_key_path: api_key_path)
```

Does anyone know what I'm do wrong?

**Comment by romk1n** [comment]
we just started to get this issue as well, is there a way to disable auto increment using fastlane?
